.circleci/config.yml

```diff
@@ -98,6 +101,32 @@ commands:
 #    - store_artifacts:
 #        path: ./build/libs
 
+  release2pypi:
+    steps:
+      - run:
+          name: Reset PyPI credentials
+          command: |
+            if [ -z "$CIRCLE_PULL_REQUEST" ]; then
+               echo "Reset PyPI Password"
+               echo "export TWINE_PASSWORD=$PYPI_PASSWORD" >> "$BASH_ENV"
+            fi
+      - run:
+          name: Release to PyPI if needed
+          command: |
+            export REGEX="^.*(emx2-pyclient).*$"
+            if [[ $COMMIT_MESSAGE_ENV =~ $REGEX ]]; then  
+              echo "RELEASE TO PYPI"
+              export $( cat build/ci.properties | xargs )
+              ./gradlew -s --no-daemon publishPython > pypi_output
+              curl -d "token=${SLACK_TOKEN}" \
+              -d "text=*<${CIRCLE_PULL_REQUEST}|Circle-CI » Molgenis » Molgenis-emx2 » PR-${CIRCLE_PULL_REQUEST##*/} #${CIRCLE_BUILD_NUM}>*
+              A new molgenis-emx2-pyclient version is released: $( grep 'View at:' pypi_output -A 1 | grep -v 'View at:' )" \
+              -d "channel=C02AZDG6QQ7" \
+              -X POST https://slack.com/api/chat.postMessage            
```

Comment:
would suggest having the slack msg as separate step , similar to other jobs

Reply:
The slack message was originally included in the other step, as it only should be posted in case of a release. But I've created a separate step for this again including the check on commit message.